client/src/components/Content.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import './style/Main.css';
+import Button from '@material-ui/core/Button';
+
+function Content({ contentValue }) {
+  const [contentText, setContentText] = useState(contentValue.slice(0, 450));
```

Comment:
Its better to trim the text with CSS `ellipsis`:
https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/
👨🏻‍🏫 - prepare 5 min explanation